e2e/multi-svc-app/front-end/main.go

```diff
@@ -69,13 +72,58 @@ func SetJobCheck(w http.ResponseWriter, req *http.Request, ps httprouter.Params)
 	w.WriteHeader(http.StatusOK)
 }
 
+// PutEFSCheck writes a file to the EFS folder in the container.
+func PutEFSCheck(w http.ResponseWriter, req *http.Request, ps httprouter.Params) {
+	efsVar := os.Getenv("COPILOT_MOUNT_POINTS")
+	copilotMountPoints := make(map[string]string)
+	if err := json.Unmarshal([]byte(efsVar), &copilotMountPoints); err != nil {
+		log.Println("Unmarshal COPILOT_MOUNT_POINTS env var FAILED")
+		w.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+	fileName := fmt.Sprintf("%s/testfile", copilotMountPoints[volumeName])
+	fileObj, err := os.Create(fileName)
+	if err != nil {
+		log.Printf("Create test file %s in EFS volume FAILED\n", fileName)
+		w.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+	// Resize file to 10M
+	if err := fileObj.Truncate(1e7); err != nil {
+		log.Printf("Resize test file %s in EFS volume FAILED\n", fileName)
+		w.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+	fileObj.Close()
+
+	// Shred file to ensure metered size increases and it's not read as sparse.
```

Comment:
I don't know what this means 😆

Reply:
Lol EFS is clever about how it reads filesize. If there isn't anything actually written to the file there's a chance that it won't show up in metering as its full size after calling Truncate. So we call `shred`, a utility which is supposed to make data recovery on magnetic drives harder by filling the file with random garbage bits. This makes sure that the file reads as 10M on disk.